oteapi/models/mappingconfig.py

```diff
@@ -1,27 +1,28 @@
 """Pydantic Mapping Configuration Data Model."""
-from typing import Annotated, Dict, List, Optional
+from typing import Dict, List, Optional, Tuple
 
-from pydantic import Field, conlist
+from pydantic import Field
 
 from oteapi.models.genericconfig import GenericConfig
 
-SemanticTriple = Annotated[list, conlist(str, min_items=3, max_items=3)]
-
 
 class MappingConfig(GenericConfig):
     """Mapping Strategy Data Configuration."""
 
     mappingType: str = Field(
-        ..., description="Type of registered mapping strategy. E.g., `mapping/demo`."
+        "mapping",
+        description="Mapping type.  Normally this is just 'mapping'.",
     )
     prefixes: Optional[Dict[str, str]] = Field(
-        None,
+        {},
```

Comment:
Either the type should be `Optional[<something>]` and the default value is `None` OR the type is `<something>` and the default value is an "empty"/default <something>.
So this should either stay `None` or the type should have `Optional[]` removed.

Reply:
So to say it is optional (with a sensible default) I should remove `Optional[]` ??

Reply:
I think removing `Optional` is less explicit, but I did as you suggested anyway.

Isn't  `Optional` simply optional in combination with `Field`? If so, I don't understand why we can add it to be explicit.

And that it depends on the value (None/non-None) whether `Optional` should be used or not sounds even more strange. Are you really sure about that?

Reply:
Well you could use `Optional[]` and still have a default non-`None` value of course - it's just not necessary and simply allows a `None` value for the attribute, which you then have to take into account when using the attribute in your code - hence my original comment. But inherently, you could do as you have done.

Reply:
If you remove the `Optional[]` and keep a default value for the attribute, you can assume the attribute is _always_  a dictionary and not worry about it being anything else - meaning standard `dict` methods will always work.